packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -111,14 +111,10 @@ const ElementHook: Hooks<VElement> = {
     create: (vnode) => {
         const {
             sel,
-            clonedElement,
             data: { ns },
             owner: { renderer },
         } = vnode;
-
-        // TODO [#1364]: supporting the ability to inject a cloned StyleElement via a vnode this is
```

Comment:
@caridy based on the description of the PR, can we close #1364 once this is merged? cc/ @pmdartus

Reply:
No, I don't think this closes that one. That one can be removed once we get node reactions in... which means we can remove the synthetic vnode for native styles... I think

Reply:
Yes agreed, let's keep this issue opened for now.